src/main/java/openfasttrack/importer/legacytag/PathConfig.java

```diff
@@ -0,0 +1,70 @@
+package openfasttrack.importer.legacytag;
+
+/*-
+ * #%L
+ * OpenFastTrack
+ * %%
+ * Copyright (C) 2016 - 2018 hamstercommunity
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+
+public class PathConfig
+{
+    private static final String REGEX_PREFIX = "regex:";
+    private static final String GLOB_PREFIX = "glob:";
```

Comment:
Out of curiosity: why two mechanisms?

Reply:
To be more flexible and `PathMatcher` already supports both.

Reply:
Ok.